developers_chamber/git_utils.py

```diff
@@ -50,6 +50,9 @@ def create_deployment_branch(environment, remote_name=None, is_hot=False):
     source_branch_name = str(repo.head.reference)
     deployment_branch_name = 'deploy-{}'.format(environment)
 
+    files_to_add = [file for file in g.diff('--name-only', '--cached').split('\n')]
+    g.stash('save')
```

Comment:
@rubickcz cached files are stashed, only HEAD is deployed and everithing is returned back after checkout